extra-scientific/bedtools/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=bedtools
+PKGSEC=science
+PKGDES="A powerful tool for genome arithmetic"
+export prefix=/usr
```

Comment:
Use `ABMK="prefix=/usr"`, but in fact this is already defined internally. That said, this line could be removed.